addons/web/static/src/views/fields/dynamic_placeholder_popover.xml

```diff
@@ -4,41 +4,41 @@
                            title="Close"
                            role="img"
                            aria-label="Close"
                            t-on-click="props.close"
                         />
                     </div>
+                    <t t-if="state.fieldName">
+                        <span t-out="state.fieldName"/>
+                        <br/>
+                        <span>or</span>
+                    </t>
                     <div class="o_model_field_selector_default_value_input mt-2">
                         <input type="text"
-                               placeholder='Type a default text or press ENTER'
-                               class="o_input"
-                               t-att-value="state.defaultValue"
-                               t-on-input="(ev) => this.setDefaultValue(ev.target.value)"
-                               t-ref="autofocus"/>
+                            placeholder="What shoud we write when the field is empty"
+                            class="o_input"
+                            t-att-value="state.defaultValue"
+                            t-on-input="(ev) => this.setDefaultValue(ev.target.value)"
+                            t-ref="autofocus"/>
                     </div>
                 </div>
-                <div class="o_model_field_selector_popover_body">
-                    <ul class="o_model_field_selector_popover_page list-unstyled mb-0 overflow-auto">
-                        <li t-attf-class="o_model_field_selector_popover_item border-bottom #{state.defaultValue === '' and 'active'}">
-                            <button class="o_model_field_selector_popover_item_name btn flex-fill border-0 rounded-0 text-start"  t-on-click="() => this.validate()">
-                                <t t-if="state.defaultValue === ''">
-                                    <div class="text-muted o_model_field_selector_item_title">Default text is used when no values are set</div>
-                                </t>
-                                <t t-else="">
-                                    <div class="text-muted o_model_field_selector_item_title">As a default text when no value are set</div>
-                                </t>
-                            </button>
-                        </li>
-                    </ul>
+                <div class="o_model_field_selector_popover_footer border-top py-1 my-2 px-2">
+                    <input type="text" class="o_input o_model_field_selector_debug"
+                        disabled="disabled" t-att-value="state.path" />
                 </div>
-                <div class="o_model_field_selector_popover_footer border-top py-1 px-2">
-                    <input type="text" class="o_input o_model_field_selector_debug" disabled="disabled" t-att-value="state.path" />
+                <div class="d-flex flex-row justify-content-around py-2">
+                    <button class="btn btn-primary" t-on-click="() => this.validate()">
```

Comment:
I never know if you need the `.bind` or if it's not allowed on `t-on-` directives
```suggestion
                    <button class="btn btn-primary" t-on-click="validate">
```